### PR TITLE
Add distributed tracing docs

### PR DESCRIPTION
**Pull request by lkysow**

### CODE REVIEW

File: website/content/docs/connect/distributed-tracing.mdx [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+1. [`envoy_extra_static_clusters_json`](/docs/connect/proxies/envoy#envoy_extra_static_clusters_json): Defines the address
+   of your tracing collector where Envoy will send its spans. In this example the URL was `collector-url:9411`.
+
+## Applying the configuration
+
+This configuration only applies when proxies are _restarted_ since it changes the _bootstrap_ config for Envoy
+which can only be applied on startup. This means you must restart all your proxies for changes to this
+config to take effect.
+
+-> On Kubernetes this is a matter of restarting your deployments, e.g. `kubectl rollout restart deploy/deploy-name`.
```
Comment (karl-cardenas-coding) [inline website/content/docs/connect/distributed-tracing.mdx:162]:
```suggestion
-> **Note:** On Kubernetes this is a matter of restarting your deployments, e.g. `kubectl rollout restart deploy/deploy-name`.
```

File: website/content/docs/connect/distributed-tracing.mdx [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+# Distributed Tracing
+
+Distributed tracing is a way to track and correlate requests across microservices. Distributed tracing must first
+be implemented in each application, it can't be added by Consul. Once implemented in your applications, adding
+distributed tracing to Consul will add the sidecar proxies as spans in the request path.
+
+## Application Changes
+
+Consul alone can't implement distributed tracing for your applications. Each application must propagate the required
+headers. Typically this is done using a tracing library.
```
Comment (karl-cardenas-coding) [inline website/content/docs/connect/distributed-tracing.mdx:17]:
```suggestion
Consul alone cannot implement distributed tracing for your applications. Each application must propagate the required
headers. Typically this is done using a tracing library.
```

Comment (karl-cardenas-coding) [inline website/content/docs/connect/distributed-tracing.mdx:17]: @lkysow can you list some libraries you would recommend? That way can help the reader by stating `Typically this is done using a tracing library such as ......`

File: website/content/docs/connect/distributed-tracing.mdx [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+layout: docs
+page_title: Distributed Tracing
+description: >-
+  Distributed tracing is a way to track and correlate requests across microservices.
+---
+
+# Distributed Tracing
+
+Distributed tracing is a way to track and correlate requests across microservices. Distributed tracing must first
+be implemented in each application, it can't be added by Consul. Once implemented in your applications, adding
```
Comment (karl-cardenas-coding) [inline website/content/docs/connect/distributed-tracing.mdx:11]:
```suggestion
be implemented in each application, it cannot be added by Consul. Once implemented in your applications, adding
```

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/546340.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 31a436bf82520424798109ad4833c9cde7381a68 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 31a436bf82520424798109ad4833c9cde7381a68 onto `release/1.11.x` succeeded!

**Pull request merged**